middleware/logger.go

```diff
@@ -34,7 +64,7 @@ func Logger() echo.MiddlewareFunc {
 				code = color.Cyan(n)
 			}
 
-			log.Printf("%s %s %s %s %d", method, path, code, stop.Sub(start), size)
+			log.Printf("%s %s %s %s %s %d", parseIP(remoteAddr), method, path, code, stop.Sub(start), size)
```

Comment:
Sure, we could, lots of times I find myself proxy-ing from another web server (nginx, apache), or a load balancer, and sometimes using headers can be helpful in those cases.  Otherwise it will look like the request is coming from the proxy host or load balancer instead of who it is really coming from.